app/views/lists/show.html.erb

```diff
@@ -1,21 +1,26 @@
-<h1><%= @list.name %></h1>
-
+<h2 class= "list-title"><%= @list.name %></h2>
+<div class="list-nav">
+  created by <%= link_to @list.user.username, user_profile_path(@list.user) %>
+   - <%= link_to 'Edit', edit_user_list_path(@list.user, @list) %>
+  <% if @list.user == current_user %>
+     - <%= link_to 'Delete List', destroy_user_list_path(@list.user, @list), data: { confirm: 'Are you sure?' } %>
+  <% end %>
+</div>
 <p>
-  <strong>Description:</strong>
-  <%= @list.description %>
-</p>
-
-<p>
-  <strong>Tags:</strong>
-  <%= @list.tag_list %>
+  <% @list.tag_list.each do |tag| %>
+    <a class="btn btn-default btn-xs"><%= tag %></a>
+  <% end %>
 </p>
+<strong>Description</strong>
+<%= simple_format(@list.description) %>
```

Comment:
This should have the same sanitize options that the other uses of simple_format do